src/nvim/normal.c

```diff
@@ -3378,6 +3378,9 @@ static bool nv_screengo(oparg_T *oap, int dir, long dist)
   col_off2 = col_off1 - curwin_col_off2();
   width1 = curwin->w_width - col_off1;
   width2 = curwin->w_width - col_off2;
+  if (width2 == 0) {
+      width2 = 1; /* avoid divide by zero */
```

Comment:
Style: two space indent, `//` comment style with capitalisation and punctuation (`// Avoid divide by zero.`), and as far as I know, put comments on a line of their own.